project/AppDependencies.scala

```diff
@@ -24,7 +24,6 @@ object AppDependencies {
     "uk.gov.hmrc"       %% s"bootstrap-test-$playVersion" % bootstrapVersion % Test,
     "org.scalatestplus" %% "selenium-4-21"                % "3.2.19.0"       % Test,
     "org.scalatestplus" %% "mockito-3-4"                  % "3.2.10.0"       % Test,
-    "org.jsoup"          % "jsoup"                        % "1.17.2"         % Test,
-    "uk.gov.hmrc"       %% "ui-test-runner"               % "0.35.0"         % Test
```

Comment:
Do we have any test code that is not used? or it was just a library that is not necessary?

Reply:
@PatrykRudnicki we used to have UI tests within the service repo, but now we have moved them out into their own repo. `ui-test-runner` is no longer needed within the service as UI tests are now here:
https://github.com/hmrc/accessibility-statement-frontend-ui-tests